src/page/ConfigPageLayout/tabs/SecretsManagementTab/SecretCard.test.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { SecretWithMetadata } from './types';
+import { formatDate } from 'utils';
+
+import { SecretCard } from './SecretCard';
+
+// Mock the formatDate utility
+jest.mock('utils', () => ({
+  formatDate: jest.fn((date) => date.toString()),
+}));
+
+const writeText = jest.fn();
+
+Object.assign(navigator, {
+  clipboard: {
+    writeText,
+  },
+});
+
+Object.defineProperty(window, 'isSecureContext', {
+  value: true,
+});
+
+describe('SecretCard', () => {
+  const mockSecret: SecretWithMetadata = {
+    uuid: 'secret-123',
+    name: 'Test Secret',
+    description: 'Test Description',
+    created_at: Number(new Date('2024-01-01')),
+    created_by: 'test-user',
+    labels: [
+      { name: 'env', value: 'prod' },
+      { name: 'type', value: 'api-key' },
+    ],
+    modified_at: Number(new Date('2024-01-02')),
+    org_id: 1,
+    stack_id: 1,
+  };
```

Comment:
Why not use the a secret from `MOCKED_SECRETS` here instead?

Reply:
Old stuff - Will replace with mocks 👍🏻